src/components/calculator.js

```diff
@@ -0,0 +1,32 @@
+import './calculator.css';
+
+function Calculator() {
```

Comment:
- [ ] It would be better if your `Calculator` component is class-based as one of the requirements. Kindly fix this minor issue. :+1: :smile:

Reply:
I did the change required but it throw the linter error
![image](https://user-images.githubusercontent.com/98621809/164043382-749ed10d-a072-40b7-bdd7-925de6d19d5e.png)

I know why react throw that error and is because the component don't work with states, if you know how to fix that linter error I appreciated it to talk me about that.

but the project works perfectly
![image](https://user-images.githubusercontent.com/98621809/164043876-f078d21a-d42f-414f-bd9b-4862839346ad.png)

The project run with all the requirements

Cheers.